### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by Baksha2008**

### CODE REVIEW

File: src/constants/routs.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const ROUTS = {
```
Comment (dimagreat) [inline src/constants/routs.ts]: `ROUTES`

Comment (rumbaa01) [inline src/constants/routs.ts]: let's name routes

File: src/database/database.providers.ts [hunk 2]
```diff
@@ -1,12 +1,12 @@
 import * as mongoose from "mongoose";
+import * as config from "config";
+
+const { path } = config.get("DBRoute");
 
 export const databaseProviders = [
   {
     provide: "DbConnectionToken",
```
Comment (dimagreat) [inline src/database/database.providers.ts]: To constants

File: src/constants/statusMessage.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+export const STATUS_MESSAGE = {
+  userAlredyExists: "This user already exists",
+  unauthorized: "UNAUTHORIZED",
```
Comment (dimagreat) [inline src/constants/statusMessage.ts]: Let's use same style for all messages

File: src/main.ts [hunk 4]
```diff
@@ -1,8 +1,14 @@
 import { NestFactory } from "@nestjs/core";
+import * as config from "config";
+
 import { AppModule } from "./app.module";
 
+import { IConfig } from "../types/config/config";
+
 async function bootstrap() {
+  const { port }: IConfig["appConfig"] = config.get("appConfig");
```
Comment (dimagreat) [inline src/main.ts]: 'appConfig' to constants

File: types/config/config.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+export interface IConfig {
```
Comment (dimagreat) [inline types/config/config.ts]: We don't need separate folder for each typing

File: src/users/constants/constatnts.ts [hunk 6]
```diff
@@ -0,0 +1,3 @@
+export const USER_MODEL_TOKEN = "UserModelToken";
```
Comment (dimagreat) [inline src/users/constants/constatnts.ts]: Rename file to `dbProviders`

File: src/pills/pills.controller.ts [hunk 7]
```diff
@@ -10,13 +10,18 @@ import {
...
 import { CreatePillDto } from "./dto/pills.dto";
-import { Response } from "express";
-import { AuthGuard } from "@nestjs/passport";
 
-@Controller("pills")
+import { PILLS } from "../constants/collection";
+import { ROUTS } from "../constants/routs";
+import { STATUS_MESSAGE } from "../constants/statusMessage";
+
+@Controller(PILLS)
```
Comment (rumbaa01) [inline src/pills/pills.controller.ts]: let's name it PILLS_ROUTE

File: src/users/users.module.ts [hunk 8]
```diff
@@ -12,9 +14,9 @@ import { usersProviders } from "./users.providers";
     DatabaseModule,
     PassportModule.register({ defaultStrategy: "jwt" }),
     JwtModule.register({
-      secretOrPrivateKey: "user",
+      secretOrPrivateKey: config.get("expireTime"),
       signOptions: {
-        expiresIn: 3600
+        expiresIn: config.get("jwtConf")
```
Comment (rumbaa01) [inline src/users/users.module.ts]: keys?

File: src/users/users.module.ts [hunk 9]
```diff
@@ -12,9 +14,9 @@ import { usersProviders } from "./users.providers";
     DatabaseModule,
     PassportModule.register({ defaultStrategy: "jwt" }),
     JwtModule.register({
-      secretOrPrivateKey: "user",
+      secretOrPrivateKey: config.get("expireTime"),
```
Comment (rumbaa01) [inline src/users/users.module.ts]: keys?

**Pull request merged**